src/features/click-to-load.js

```diff
@@ -801,43 +858,6 @@ function hideInfoTextIfNarrowPlaceholder (shadowRoot, placeholder, narrowWidth)
     }
 }
 
-/**
- * Replace the blocked CTL elements on the page with placeholders.
- * @param {HTMLElement} [targetElement]
- *   If specified, only this element will be replaced (assuming it matches
- *   one of the expected CSS selectors). If omitted, all matching elements
- *   in the document will be replaced instead.
- */
-async function replaceClickToLoadElements (targetElement) {
```

Comment:
I moved this function inside `ClickToLoad` class to have access to the its context and `this.platform` when creating a new  `DuckWidget`